content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/clientlibs/site/js/image.js

```diff
@@ -196,6 +198,11 @@
             // temporary document avoids requesting the image before removing its src
             var temporaryDocument = parser.parseFromString(markup, "text/html");
             var imageElement = temporaryDocument.querySelector(selectors.image);
+            // Remove src if the src image referred is not a temp image file
+            var srcAttribute = imageElement.getAttribute("src");
+            if (srcAttribute && srcAttribute.indexOf(TEMP_DESIGN_PATH) == -1) {
```

Comment:
@shivendra096 - this clause is redundant, the attribute is still removed on the next line